modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -362,6 +412,16 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
                                 " Visit Label and CandID."
                             );
                     }
+                    // Basic fields have been validated, check user permisions
+                    $session = \TimePoint::singleton($candidate['SessionID']);
```

Comment:
This is not going to work: `TimePoint::singleton` takes a `SessionID` as argument.

Reply:
🤦